webpack.config.client.js

```diff
@@ -56,7 +62,11 @@ module.exports = ({ resolvePath, IS_CI, IS_PROD, START_DEV_SERVER }) => {
         : prodPublicPath,
     },
     optimization: {
-      minimizer: [new TerserPlugin()],
+      minimizer: [
```

Comment:
Maybe add a brief comment here explaining what this does like other settings in this config file?